pkgdb/tests/setup_suite.bash

```diff
@@ -16,17 +16,18 @@ bats_require_minimum_version '1.5.0';
 
 # Locate repository root.
 repo_root_setup() {
-  if [[ -z "${REPO_ROOT:-}" ]]; then
-    if [[ -d "$PWD/.git" ]] && [[ -d "$PWD/tests" ]]; then
-      REPO_ROOT="$PWD";
-    else
-      REPO_ROOT="$( git rev-parse --show-toplevel||:; )";
-    fi
-    if [[ -z "$REPO_ROOT" ]] && [[ -d "$PWD/tests" ]]; then
-      REPO_ROOT="$PWD";
-    fi
-  fi
-  export REPO_ROOT;
+  # if [[ -z "${REPO_ROOT:-}" ]]; then
+  #   if [[ -d "$PWD/.git" ]] && [[ -d "$PWD/tests" ]]; then
+  #     REPO_ROOT="$PWD";
+  #   else
+  #     REPO_ROOT="$( git rev-parse --show-toplevel||:; )";
+  #   fi
+  #   if [[ -z "$REPO_ROOT" ]] && [[ -d "$PWD/tests" ]]; then
+  #     REPO_ROOT="$PWD";
+  #   fi
+  # fi
+  # export REPO_ROOT;
```

Comment:
```suggestion
```
I believe using the test dir here as the source of truth is more reliable than trying to find a .git dir or test dir based on $PWD.

pinging @garbas @aakropotkin as this might have implications for test scripts we are using. (Note this is in pkgdb, so `flox-tests` should be unaffected).
That said, we now have distinct bats test suites in flox...
Maybe its worth to look at unifying at least common setup functions.

Reply:
Looks good to me.

There are bunch of things we should reuse now that we have everything in one repo.